packages/astro/src/core/config/schemas/base.ts

```diff
@@ -280,6 +284,12 @@ export const AstroConfigSchema = z.object({
 				.default(ASTRO_CONFIG_DEFAULTS.image.experimentalDefaultStyles),
 		})
 		.default(ASTRO_CONFIG_DEFAULTS.image),
+	svg: z
+		.object({
+			optimize: z.boolean().default(true),
+			svgoConfig: z.record(z.any()).optional(),
```

Comment:
I think we should offer at least proper typing here even if we don’t do full schema validation. For example, if a type is available for the SVGO options, we could use something like:
```js
			svgoConfig: z.custom<SVGOOptions>((value) => value && typeof value === 'object').optional(),
```
That way users should get a nice editor experience.